docs/src/fock.md

```diff
@@ -0,0 +1,147 @@
+# Fock States
+
+```@meta
+DocTestSetup = quote
+    using QuantumSymbolics, QuantumOptics
+end
+```
+
+In this section, we describe symbolic representations of Fock states in QuantumSymbolics, which can be numerically translated to `QuantumOptics.jl`. A Fock state is the number state representation $|n\rangle$ of a system with $n$ particles. 
```

Comment:
The definition of Fock state here is misleading. Something like "the number of excitation quanta of a single quantum harmonic oscillator" would be better. We have exactly one system here.